src/Formatter/ClassNameFormatter.php

```diff
@@ -26,6 +26,11 @@ public function commandHandled($command)
         return 'Command succeeded: ' . get_class($command);
     }
 
+    public function propertiesUsed($command)
```

Comment:
Works for me. Probably needs a decent docblock though, for folks who aren't up on monolog/PSR-3 practices though :)